app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -359,6 +359,10 @@ class LuciBuildService {
     userData['repo_owner'] = slug.owner;
     userData['repo_name'] = slug.name;
     userData['user_agent'] = 'flutter-cocoon';
+    // Retries were used to auto re-run builds when they failed with infra
+    // failure. Now with github checks api support automated retries won't be
+    // needed anymore and will be removed:
+    // https://github.com/flutter/flutter/issues/60942.
```

Comment:
Nit: you may want to follow the TODO format.

Reply:
Done!